shared/versions.yml

```diff
@@ -87,36 +87,36 @@ couchdb:
     tag: 2.3.1
 dataloader:
   upstream:
-    repository: gpii/universal
-    tag: 20200330152413-ad461d9
+    repository: herrclown/vagrant-universal
+    tag: latest
   generated:
-    repository: gcr.io/gpii-common-prd/gpii__universal
```

Comment:
We should keep using the gpii/universal official repository instead. Is this possible @klown ?

Reply:
Hi @amatas 
We won't be using this version of `versions.yml` because [my changes to gpii-universal](https://github.com/GPII/universal/pull/840) have not been merged into master yet; and `versions.yml` is using my docker image for the `flowmanager`, `preferences`, `dataloader`, `flushtokens`, and `productiontests` components.  When the changes have been merged into master, I will update `versions.yml` to reference the `gpii/universal` repository with the correct tag.

This PR is not ready to merge into gpii-infra's master branch.  It's just a way of testing the changes in my dev cluster, in concert with @stepanstipl's changes to istio.

Reply:
I would mark this PR as "work-in-progress", but I don't have the authority to add labels.

Reply:
I did it, thanks @klown